ui/ducks/locale/locale.js

```diff
@@ -13,7 +14,25 @@ export default function reduceLocaleMessages(state = {}, { type, payload }) {
   }
 }
 
-export const getCurrentLocale = (state) => state.localeMessages.currentLocale;
+/**
+ * @param state
+ * @returns {string} one of the codes in file://./../../../app/_locales/index.json.
```

Comment:
Should we be explicit in both these method's JSDoc that it's the user's selected locale that is returned?

Reply:
Done.